BraveRewards/Ledger/BATBraveLedger.mm

```diff
@@ -452,6 +497,16 @@ - (void)onGrant:(ledger::Result)result grant:(const ledger::Grant &)grant
 {
   if (result == ledger::LEDGER_OK) {
     [self.mPendingGrants addObject:[[BATGrant alloc] initWithGrant:grant]];
+    
+    bool isUGP = grant.type == "ugp";
```

Comment:
Isn't there a constant for `"ugp"` type? 
If not, I would extract it to constant somewhere, we use it in two places

Reply:
There's no constant in the ledger lib, but I can extract them at a least here